test/test.js

```diff
@@ -1020,7 +1068,8 @@ suite('GaiaHeader', function() {
   });
 
   test('it clears pending async `setTitleStyles`', function(done) {
-    this.dom.innerHTML = '<gaia-header action="menu"><h1>Title</h1></gaia-header>';
+    this.dom.innerHTML = '<gaia-header action="menu"><h1>Title</h1>' +
+      '</gaia-header>';
     var el = this.dom.firstElementChild;
 
     el.runFontFit();
```

Comment:
Overall I think the line wrapping in tests to conform to 80 char maxlen just makes things harder to read and maintain. I vote to opt out of this particular rule in test files.